zerver/views/auth.py

```diff
@@ -760,7 +760,7 @@ def login_page(
     # logged-in app.
     is_preview = "preview" in request.GET
     if settings.TWO_FACTOR_AUTHENTICATION_ENABLED:
-        if request.user and request.user.is_verified():
+        if is_2fa_verified(request.user):
```

Comment:
I suppose it may be a similarly  mysterious `if` to https://chat.zulip.org/#narrow/stream/3-backend/topic/request.2Euser.20being.20nonexistent :thinking:

Reply:
Possible, it's pretty much unknown. I will add this back in just for now. We will need to investigate this anyway, but most likely in a different PR.